jetson/percep/test_protocol.md

```diff
@@ -0,0 +1,73 @@
+## Compile Test
+- [ ] ` ./jarvis build jetson/percep -o with_zed=false ar_detection=true obs_detection=true perception_debug=true `
+  - [ ] No compiler errors or warnings
+- [ ] ` ./jarvis build jetson/percep -o with_zed=false ar_detection=true obs_detection=true perception_debug=false `
+  - [ ] No compiler errors or warnings
+- [ ] ` ./jarvis build jetson/percep -o with_zed=true ar_detection=true obs_detection=true perception_debug=false `
+  - [ ] No compiler errors, only ZED warnings
+- [ ] ` ./jarvis build jetson/percep -o with_zed=true ar_detection=true obs_detection=true perception_debug=true `
+   - [ ] No compiler errors, only ZED warnings
+
+## Offline Debug Test
+- [ ] `./jarvis build jetson/percep -o with_zed=false ar_detection=true obs_detection=true perception_debug=true`
+- [ ] Run code against test1 image set
```

Comment:
Might want to say 'Run code against test1 image set, located in home/uniquename/mrover-workspace/perception-files/test1'

Reply:
Since all of our test images are located in the perception-files repository I'll mention that it's located in perception-files. I'd rather not include the complete address since it varies between dev envs